### DISCUSSION TITLE
能帮我看看我的音乐哪里是PCL接取不了的吗

### DISCUSSION THREAD

**Discussion by kong-que-dnf123** [Meloong-Git/PCL#2733] [提问] [answered]
![image](https://github.com/Hex-Dragon/PCL2/assets/120994373/9f9e31c7-8aae-4ab6-a0e2-f4a6914a2012)

_github不让上传音乐就压缩就压缩了......_
[ink sans 1~4.zip](https://github.com/Hex-Dragon/PCL2/files/12775792/ink.sans.1.4.zip)

**Comment by lactobionicAcid** [comment, marked as answer]
[ink sans 1~4 fixed.zip](https://github.com/Hex-Dragon/PCL2/files/12776258/ink.sans.1.4.fixed.zip)
把你的音频修了一下，现在应该行了

**Reply by kong-que-dnf123** [reply]
谢了
